test/unit/symbol-parser.spec.ts

```diff
@@ -0,0 +1,162 @@
+import * as elf from "elfy";
+import * as fs from "fs";
+import { join as pathJoin } from "path";
+
+import { FileContent } from "../../lib/extractor/types";
+import * as symbols from "../../lib/go-parser/symbols";
+import { Elf } from "../../lib/go-parser/types";
+
+// contains all test cases for the PCLN Tab / Symbol table parser.
```

Comment:
Not sure what to add there without duplicating information or having it go out of date. What would you expect on the notion page?

Reply:
I'm not worried about duplicating information, I just want a link we can send on call/support/customer success when they have a question about the support to cgo and stripped binaries. What do you think about adding it to the existing spike document?

Reply:
I think I'll add something to the docs once this is merged, so that I can link both PRs as well. Or do you think I need to document something already?